### PR TITLE
[AUT-3116] Redesigned Makefile to allow specify path to ACP charts files

### PR DESCRIPTION
**Pull request by lisocki**
## Description

Redesigned `Makefile` to allow specify path to ACP chart files (with `ACP_CHARTS_PATH` env variable).
Also cleanded code in this file.

## Type of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Tests (extending the test suite)
- [x] Refactor (internal improvement that doesn't change product functionality)
- [ ] Other (if none of the other choices apply)

### CODE REVIEW

**Review by jkotiuk — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -1,136 +1,183 @@
...
+	./istio-1.9.3/bin/istioctl install --filename ./config/ce-istio-profile.yaml --skip-confirmation
 	kubectl label namespace default istio-injection=enabled
-	rm -rf ./istio-1.9.3
+	rm --recursive --force ./istio-1.9.3
+
+wait-acp:
+	kubectl wait deploy/acp \
+		--for condition=available \
+		--namespace acp-system \
+		--timeout 5m
```
Comment (ppodbielski-cloudentity) [inline Makefile:81]: why decrease to 5 min?

File: Makefile [hunk 2]
```diff
@@ -1,136 +1,183 @@
...
+		--name acp \
+		--config ./config/kind-cluster-config.yaml
 
 prepare-helm:
 	helm repo add acp https://charts.cloudentity.io
 	helm repo add ingress-nginx https://kubernetes.github.io/ingress-nginx
 	helm repo add jetstack https://charts.jetstack.io
 	helm repo update
+ifdef ACP_CHARTS_PATH
+	yq eval '(.dependencies[]|select(.name == "acp").repository) |= "file://../acp"' "${ACP_CHARTS_PATH}"/kube-acp-stack/Chart.yaml --inplace
```
Comment (ppodbielski-cloudentity) [inline Makefile:35]: why not `KUBE_ACP_STACK_CHART`?

File: Makefile (hunk 1 above)
Comment (lisocki) [reply Makefile:81]: It is sufficient time to deploy acp only.

File: Makefile (hunk 2 above)
Comment (lisocki) [reply Makefile:35]: You are right, it should be `KUBE_ACP_STACK_CHART` here.

Comment (lisocki) [reply Makefile]: Done

**Review by ppodbielski-cloudentity — APPROVED** [review]

**Pull request merged**